controllers/fcmToken.js

```diff
@@ -0,0 +1,22 @@
+const { saveFcmToken } = require("../models/fcmToken");
+
+/**
+ * Route used to get the health status of teh server
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const fcmTokenController = (req, res) => {
+  const { fcmToken } = req.body;
+
+  // Save the FCM token to your database or perform other necessary actions.
+  // You can associate the token with a user or device, for example.
+
+  const fcmTokenId = saveFcmToken({ userId: req.userData.id, fcmToken });
```

Comment:
Sending sensitive information like authentication tokens in a response, especially in a client-facing response like a POST response, is generally not recommended. It poses security risks, as the token might be exposed to unintended parties or stored insecurely on the client side.